metrics/collectors/versions/versions_collector.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Canonical Ltd
+
+import datetime
+import urllib.request
+
+from metrics.lib.basemetric import Metric
+
+VERSIONS_STATS_URL = "https://people.canonical.com/~platform/desktop/stats/%s/" % datetime.datetime.now().strftime(
+    "%Y-%m-%d"
+)
+
+
+class VersionsMetrics(Metric):
+    def collect(self):
+        """ Collect the sponsoring queue details"""
+        data = []
+
+        known_reports_lst = []
+        with urllib.request.urlopen(VERSIONS_STATS_URL + "reports") as reports:
```

Comment:
nitpick: can we use [`urljoin`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin) to construct the URL please?

Reply:
Changed